hat/assets/js/apps/Iaso/domains/app/index.tsx

```diff
@@ -8,18 +8,23 @@ type Props = {
     userHomePage?: string;
 };
 
+const dashboardBasename = '/dashboard';
+
 const App: FunctionComponent<Props> = ({ userHomePage }) => {
-    const { routes, isLoadingRoutes } = useRoutes(
+    const { nonDashboardRoutes, routes, isLoadingRoutes } = useRoutes(
         userHomePage && userHomePage !== '' ? userHomePage : undefined,
     );
     useSnackBars();
-    return (
-        <>
-            {isLoadingRoutes && <LoadingSpinner />}
-            {!isLoadingRoutes && (
-                <BrowserRouter basename="/dashboard">{routes}</BrowserRouter>
-            )}
-        </>
+
+    const isDashboardPath =
+        window.location.pathname.includes(dashboardBasename);
+    if (isLoadingRoutes) {
+        return <LoadingSpinner />;
+    }
+    return isDashboardPath ? (
+        <BrowserRouter basename={dashboardBasename}>{routes}</BrowserRouter>
+    ) : (
+        <BrowserRouter>{nonDashboardRoutes}</BrowserRouter>
```

Comment:
I would prefer that iaso routes cannot be accessed at all from the public registry, but it does the job